File: StudentTracker/StudentTracker/Entities/Student.cs
namespace StudentTracker.Entities
{
    public class Student
    {
        public Guid Id { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Email { get; set; }
        public long? IdNumber { get; set; }
        public string? ProfileImage { get; set; }
    }
}

File: Student-Tracker-App/src/models/student.ts
import { Enrolment } from "./enrolment";

export interface Student {
    id: string; // Guid in C# maps to string in TS
    firstName?: string;
    lastName?: string;
    email?: string;
    idNumber?: number; // long? in C# maps to number | undefined in TS
    profileImage?: string;
    enrolments?:Enrolment[]
  }